src/bridge/_global.cpp

```diff
@@ -25,3 +25,4 @@ DBGGETBPLIST _dbg_getbplist;
 DBGDBGCMDEXECDIRECT _dbg_dbgcmddirectexec;
 DBGGETBRANCHDESTINATION _dbg_getbranchdestination;
 DBGSENDMESSAGE _dbg_sendmessage;
+DBGMEMISCODEPAGE _dbg_memiscodepage;
```

Comment:
Sorry, I cannot accept the pull request because of adding new exports. There were various mechanisms put in place to prevent this, one of which is `DbgFunctions` and the other one is `_dbg_sendmessage`.